ci/dockerfiles/ansible-e2e-hybrid.Dockerfile

```diff
@@ -22,12 +22,24 @@ ENV OPERATOR=/usr/local/bin/ansible-operator \
 # Install python dependencies
 # Ensure fresh metadata rather than cached metadata in the base by running
 # yum clean all && rm -rf /var/yum/cache/* first
-RUN yum clean all && rm -rf /var/cache/yum/* \
- && (yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm || true) \
- && yum -y update \
- && yum install -y python36-devel python36-pip gcc \
- # Install inotify-tools. Note: rpm -i will install the rpm in the registry for allow yum install it.
- && curl -O https://rpmfind.net/linux/fedora/linux/releases/30/Everything/x86_64/os/Packages/i/inotify-tools-3.14-16.fc30.x86_64.rpm \
+RUN yum clean all && rm -rf /var/cache/yum/*
+
+# todo; remove ubi7 after CI be updated with images
+# ubi7
+RUN if $(cat /etc/redhat-release | grep --quiet 'release 7'); then (yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm || true); fi
+# ubi8
+RUN if $(cat /etc/redhat-release | grep --quiet 'release 8'); then (yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm || true); fi
+
+RUN yum -y update \
+ && yum install -y python36-devel gcc
+
+# ubi7
+RUN if $(cat /etc/redhat-release | grep --quiet 'release 7'); then (yum install -y python36-pip || true); fi
+# ubi8
+RUN if $(cat /etc/redhat-release | grep --quiet 'release 8'); then (yum install -y python3-pip || true); fi
+
+# Install inotify-tools. Note: rpm -i will install the rpm in the registry for allow yum install it.
+RUN curl -O https://rpmfind.net/linux/fedora/linux/releases/30/Everything/x86_64/os/Packages/i/inotify-tools-3.14-16.fc30.x86_64.rpm \
```

Comment:
I _think_ we should still be able to use the `\ <newline> &&` syntax to have this entire set of stuff happen in a single docker layer.

Reply:
We will remove it in the next PR. It is just to check the CI.

Reply:
We will remove all ifs in the next PR. It is just for don't break the CI. Then, all will come back to be just 1 layer. Is it make sense?

Reply:
I don't think there's a huge rush on this, and I think we should be more diligent than we normally would be because the ansible base image build is a pretty regular source of issues in CI.

I don't have a major problem with multi-layer vs. single layer since that shouldn't make a difference in the resulting image.

However, I'm a bit confused with some of the additions and changes for ubi8 and want to make sure that we get things right from the start.

1. For ubi7, my impression is that we are installing the EPEL repo because it contains `inotify-tools`, so if we're using EPEL, we should be installing `inotify-tools` with `yum install -y inotify-tools`, not by downloading and manually installing.
2. For ubi8, it appears that EPEL does not contain `inotify-tools`, so I don't think we should be installing EPEL in ubi8. In this case, I'm curious why it disappeared, and if it is now present in another repo, or if it has been removed entirely from RHEL/UBI8. I'd rather that we not download from `rpmfind.net` if there's an official place that `inotify-tools` has moved. And if it is totally removed from official RHEL/UBI8 repos, we should have a comment (and ideally a link to discussion about its removal) in the Dockerfile.

Reply:
Hi @joelanford,

Thank you for your review and inputs. 

Just to clarify my idea was we do it now just to pass in the prow CI and we are able to get the ubi8 then we could improve. 

* Regards the epel I think is a good spot and we can not use it. 
* Regards the inotify-tools it was removed from ubi8 and we have for it inotify3-tools. So, let's try to use it now and avoid do the download. Note that if we run locally the docker image with it all is installed so let's cross the fingers for it be == and pass in all ci tests as well.